### ISSUE TITLE
Hot code reload

### ISSUE THREAD

**Issue by nonameplum** [crossroadlabs/Express#8]
Please, like Play framework provide hot code reload, this feature make your project one of the most valuable swift web framework on the market

I love your idea behind to get best parts of such great frameworks!

**Comment by dileping** [comment]
Hi @nonameplum,

first of all thanks for kind words and sharing our vision! This helps us understand we are on the right path. As for hot code reload we are actively discussing it inside the team already. I will add it to the roadmap on the README. Will definitely be there!

**Comment by nonameplum** [comment]
Yes, I think that Play and Express are one of the most friendly and well written web frameworks. You could also look on ideas behind Vert.x. Do not forget about modularisation, community can start work for you and create new modules/extensions like minification, concatenation, custom authorisation modules, auto prefixes, compress etc.

**Comment by dileping** [comment]
@nonameplum Never tried vertex, unfortunately, but first impression is good. It feels like it follows the same patterns as Play and Express.js. Thanks!

As for the modularisation, we do it :) I think I should write a small guide on extending Express. Thanks for pointing to it.
